src/components/Home.js

```diff
@@ -105,6 +133,89 @@ function Home() {
           </div>
         </Carousel.Item>
       </Carousel>
+
+      </div>
+      :
+      <header className="App-header-home">
+          <img src={extension} className="App-logo-home" alt="logo" />
+          <div id="slogan">
+            <TextLoop>
+                <span>Support <span id="smile">Global Health</span></span>
+                <span>Stand with <span id="smile">Racial Justice</span></span>
+                <span>Provide <span id="smile">Water &amp; Food Security</span></span>
+                <span>Empower <span id="smile">Women</span></span>
+            </TextLoop>{" "}
+            <div className="subtitle-home">Donate <span id="emphasize">without spending extra</span> when shopping online.</div>
+            <Button variant="outline-secondary btn-round-header" size="lg" href="https://tiny.cc/soulsmile-extension">Add to chrome</Button>{' '}
+          </div><br/>       
+      </header>
+      } {/* TFAC Ternary Operator 2 End*/}
+
+      {/* TFAC Ternary Operator 3 Start*/}
+      {collab ?
```

Comment:
I think this section would be clearer and less repetitive if you created a few separate variables for commonly used elements in the home page (ex: OluMilyCarousel,  CausesTextLoop, etc.)!

Reply:
Updated!